package.json

```diff
@@ -6,24 +6,22 @@
     "packages/*"
   ],
   "dependencies": {
-    "husky": "8.0.1",
-    "lerna": "5.5.0"
+    "husky": "8.0.1"
   },
   "devDependencies": {
     "lerna-update-wizard": "1.1.0"
   },
   "scripts": {
-    "build": "lerna run build --ignore datagateway-common",
-    "lint": "lerna run lint:js -- --max-warnings=0",
-    "tsc": "lerna run tsc",
+    "build": "yarn workspaces foreach --verbose --parallel --exclude datagateway-common run build",
+    "lint": "yarn workspaces foreach --verbose --parallel run lint:js --max-warnings=0",
```

Comment:
- Can we add `--interlaced`?
- If we add `--jobs 3` to this, it may run faster? I think we only need 3 instead of 4 because common needs to be linted first before the other packages